docs/resources/server.md

```diff
@@ -169,6 +169,9 @@ The following arguments are supported:
   * `network_interface_no` - (Required) If you want to add a network interface that you created yourself, set the network interface ID.
   * `order` - (Required) Sets the order of network interfaces to be assigned to the server to create. The unit name (eth0, eth1, etc.) is determined in that order. There must be one primary network interface. If you set `0`, network interface is set by default. You can assign up to three network interfaces.
 * `is_encrypted_base_block_storage_volume` - (Optional) you can set whether to encrypt basic block storage if server image is RHV. Default `false`.
+* `block_device_partition_list` - (Optional) You can set up partitions when creating BareMetal servers. Partitions may not be supported, depending on the server specifications.
+  * `mount_point` - (Required) Enter a mount point that starts with the "/" (root) path. The first mount must be a "/" (root) partition. Only lowercase English letters and numbers are allowed for names under "/" (root), and must start with a lowercase English letter. Depending on the OS type, certain keywords such as /root, /bin, and /dev may not be available.
```

Comment:
(trivial) I'd like to suggest describing in the form of nouns rather than imperative statements. (sometimes feel like auto-translated from other languages).
```suggestion
* `block_device_partition_list` - (Optional) List of block device partitions for the BareMetal server. Partitions may not be supported, depending on the server specifications.
  * `mount_point` - (Required) Mount point. It starts with the "/" (root) path. The first mount must be a "/" (root) partition. Only lowercase English letters and numbers are allowed for names under "/" (root), and must start with a lowercase English letter. Depending on the OS type, certain keywords such as /root, /bin, and /dev may not be available.
```